silverback/exceptions.py

```diff
@@ -3,21 +3,23 @@
 from ape.exceptions import ApeException
 from ape.logging import logger
 
+from .types import TaskType
+
 
 class ImportFromStringError(Exception):
     pass
 
 
-class DuplicateHandlerError(Exception):
```

Comment:
Would any downstream usage be importing / catching these exceptions? It would be breaking if yess.

Reply:
The errors that subclass `Exception` are just usage errors, but maybe there's a better way to demonstrate that